core/kt-osrd-rjs-parser/src/main/kotlin/fr/sncf/osrd/RawInfraRJSParser.kt

```diff
@@ -754,12 +766,33 @@ fun parseRJSInfra(rjsInfra: RJSInfra): RawInfra {
         for (opPart in operationalPoint.parts) {
             // ignore duplicates
             if (distinctParts.contains(opPart)) continue
-
             distinctParts.add(opPart)
+
+            val operationalPointId = operationalPoint.id
+            val trackSectionName = opPart.track
+            val trackSectionOffset = Offset<TrackSection>(opPart.position.meters)
+            val props = mutableMapOf<String, String>()
+            props["trackSectionName"] = trackSectionName
+            props["position"] = opPart.position.toString()
```

Comment:
why is that needed? isn't it already available using other APIs?

Reply:
`trackSectionName` is indeed unneeded. However, we want to keep `position` as a double, so i need to keep it there.